src/DataLayouts/DataLayouts.jl

```diff
@@ -1481,4 +1485,52 @@ get_Nij(::IFH{S, Nij}) where {S, Nij} = Nij
 get_Nij(::IJF{S, Nij}) where {S, Nij} = Nij
 get_Nij(::IF{S, Nij}) where {S, Nij} = Nij
 
+
+"""
+    data2array(::AbstractData)
+
+Reshapes the DataLayout's parent array into a `Vector`, or (for DataLayouts with vertical levels)
+`Nv x N` matrix, where `Nv` is the number of vertical levels and `N` is the remaining dimensions.
+
+The dimensions of the resulting array are
+ - `([number of vertical nodes], number of horizontal nodes)`.
+
+Also, this assumes that `eltype(data) <: Real`.
+"""
+function data2array end
+
+data2array(data::Union{IF, IFH}) = reshape(parent(data), :)
+data2array(data::Union{IJF, IJFH}) = reshape(parent(data), :)
```

Comment:
Are these two functions being tested?

Reply:
I believe so, IIRC

Reply:
Ah, actually, no. I'll add some (there were none in ClimaAtmos)